routes/routes.js

```diff
@@ -38,5 +38,31 @@ router.post('/user/answers/question/:questionNumber', (req, res) => {
     })
 })
 
+router.get('/user/answers/question/:questionNumber', (req, res) => {  
+    let pageSize = 3; // TODO: define it in more stadarized way 
+    
+    User.findOne({ 
+        where: {id: req.query.userId}
+    }).then(user => {
+        
+        UserAnswers.findAll({
+            where: {question_number: req.params.questionNumber, user_id: user.id},
+            order: [
+                ['created_at', 'DESC']
+            ],
+            limit: pageSize
+        }).then(userAnswers => {
+            return res.status(200).json({
+                previousTimingMs: userAnswers.map(userAnswer => userAnswer.elapsed_time_ms)
+            })
```

Comment:
Since 200 is the default response code, you don't need to specify it unless the code would be confusing otherwise.

Reply:
Sometimes (when I skip typing the answer) it returns 304.